package.json

```diff
@@ -1,8 +1,14 @@
 {
   "name": "sassjs-loader",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
If the we're now supporting Webpack 4, then presumably it no longer works with Webpack 1. So this should be a major bump, because it's breaking backwards compat.

Reply:
Also, you should change the `webpack` version in `devDependencies` to be 4, so we're testing with the version that we want to support.